test/JDBC/input/collations/verify_mapping_for_remove_accents.mix

```
@@ -0,0 +1,36 @@
+-- sla 100000
+-- Insert all possible UNICODE/UTF-8 valid characters
+-- tsql
+CREATE TABLE all_utf8_chars(col1 BYTEA, col2 TEXT)
+GO
+
+-- psql
+DO $$
+DECLARE
+    erro TEXT;
+    cnt INT:= 1;
+    var TEXT;
+BEGIN
+    WHILE cnt <= 1114112 LOOP
+        BEGIN
+            INSERT INTO master_dbo.all_utf8_chars VALUES (cast(chr(cnt) as bytea), chr(cnt));
+        EXCEPTION
+        WHEN others THEN
```

why we ignore error here?

---

Some integer are converted to hex value which is not valid in that encoding, so we get an error like
```ERROR:  requested character not valid for encoding: 57330```
We want to skip these errors.